### ISSUE TITLE
successful compilation generates error report instead of info report

### ISSUE THREAD

**Issue by Brodan** [klacke/yaws#247]
when compiling .erl source files, the Eshell outputs the compilation results as an "ERROR REPORT" regardless of whether or not there are errors/warnings. in the case of 0 errors and 0 warnings, the output should display in INFO REPORT instead

example output:
```
$ sudo yaws -i
...
=ERROR REPORT==== 1-Dec-2015::16:47:41 ===
Compiled "/usr/local/var/yaws/www/src/test.erl" [0 Errors - 0 Warnings]
...
```

**Comment by vinoski** [comment]
Can you provide more details about what you're doing to cause your file to be compiled?

**Comment by Brodan** [comment]
in my yaws.conf file, i've specified the following as suggested [here](http://yaws.hyber.org/yman.yaws?page=yaws.conf):

```src_dir = /usr/local/var/yaws/www/src```

test.erl looks as such:
```
-module(test).
-export([thing/1]).
thing(X) -> X.
```

**Comment by vinoski** [comment]
OK, thanks. I've reproduced the issue.

**Comment by vinoski** [comment]
Fixed in c8f1918.
